documentation/asciidoc/computers/configuration/headless.adoc

```diff
@@ -20,20 +20,18 @@ WARNING: Depending on the model and SD card, your Raspberry Pi may require up to
 With no keyboard or monitor, you need a way to xref:remote-access.adoc[remotely control] your headless Raspberry Pi. On first boot, the only option is SSH. To enable SSH on a fresh installation of Raspberry Pi OS, choose one of the following methods:
 
 * enable SSH in the Advanced Menu of Raspberry Pi Imager, then enter a username and password
-* place a file named `ssh` (no file extension) into the boot folder (`/boot/firmware/`) of the SD Card, then configure a user with `userconf.txt`
-
-IMPORTANT: To use SSH on a fresh Raspberry Pi OS installation, you must create a user account in the xref:getting-started.adoc#advanced-options[Advanced Menu] of Raspberry Pi Imager.
+* place a file named `ssh` into the root (`/`) of the SD card, then configure a user manually with `userconf.txt` following the instructions below
```

Comment:
If there's only two options, does it really need a bulleted list? :thinking:  :shrug:

Reply:
Nah, in hindsight ignore this :)